### ISSUE TITLE
Staging Review finding: Radio group error message not announced to screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#124643]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization (Aquia)
**Product:** VA Form 21-0779
**Feature:** MVP Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On Step 4 of 7 Certification of level of care, when the radio group has an error, and the user tabs back into the group, the error message is not announced to screen reader users. 

Other occurences:
- Step 5 of 7 Medicaid facility status
- Step 5 of 7 Medicaid application status
- Step 5 of 7 Medicaid status

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="272" height="405" alt="radio error" src="https://github.com/user-attachments/assets/b18fb4b5-af0e-4d87-86e1-f31c74d6a293" />

</details>

## Recommended action

Ensure that the field is identified as in error progammatically using `aria-invalid="true"` and ensure that the error message is associated to the field in error, so that screen readers announce the error when the radio has focus. 

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by LinnJS** [comment]
Fixed in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/39962)

Results

<img width="817" height="501" alt="Image" src="https://github.com/user-attachments/assets/3a5ffa26-00e2-4b19-b64d-d48530b0dc49" />

**Comment by leahkeeler** [comment]
fixed per above
